MaxText/layers.py

```diff
@@ -440,6 +440,11 @@ def query_init(*args):
     key = projection(kernel_init=self.kernel_init, name='key')(inputs_kv)
     value = projection(kernel_init=self.kernel_init, name='value')(inputs_kv)
 
+    #Apply RoPE
+    query = LLaMARotaryEmbedding(embedding_dims = self.head_dim, name='query_rotary')(query)
```

Comment:
Since we are sequence_packing our inputs, I think we should also pass `inputs_position`. You can get this information after you plumb it in the Transformer class from train.py

Reply:
Excellent suggestion - thanks Mohit! I made the change